targeted/cve-2021-22214.js

```diff
@@ -5,6 +5,14 @@
 
 var pluginid = 100024;
 
+var Sender = Java.type("org.parosproxy.paros.network.HttpSender")
```

Comment:
`HttpSender`, it's clearer if it matches the class name.

Reply:
Makes sense